tsconfig.json

```
@@ -31,7 +31,8 @@
     "noEmit": true,
     "paths": {
       // TEMPORARY: Until we can fix the "exports" in the loot-core package.json
-      "loot-core/*": ["./packages/loot-core/src/*"]
+      "loot-core/*": ["./packages/loot-core/src/*"],
+      "@hooks/*": ["./packages/desktop-client/src/hooks/*"]
```

Shifting the conversation here.

This is what I meant. If we do this:
```suggestion
      "@app/*": ["./packages/desktop-client/src/*"]
```

Then we can import all the folders from `@app` scope without needing to define each folder individually.

(nit: it would be even nicer if we could define the alias inside the `packages/desktop-client` scope instead of in global tsconfig, but your solution works too.. we can always change the configs later without updating all the improts)